rules/DeadCode/Rector/If_/RemoveDeadInstanceOfRector.php

```diff
@@ -135,12 +140,13 @@ private function processMayDeadInstanceOf(If_ $if, Instanceof_ $instanceof): ?If
             return null;
         }
 
+        $this->removeNode($if);
+
         if ($if->cond === $instanceof) {
-            $this->nodesToAddCollector->addNodesBeforeNode($if->stmts, $if);
+            return $if->stmts;
```

Comment:
`if->stmts` need to be checked to have at least 1 stmt before returned:

```suggestion
            return count($if->stmts) > 0 ? $if->stmts : $if;
```

Reply:
https://github.com/rectorphp/rector-src/pull/2252 to solve error because of it

```
ime: 00:02.010, Memory: 100.50 MB

There was 1 error:

1) Rector\Tests\DeadCode\Rector\If_\RemoveDeadInstanceOfRector\RemoveDeadInstanceOfRectorTest::test with data set #14 (Symplify\SmartFileSystem\SmartFileInfo Object (...))
Rector\Core\Exception\ShouldNotHappenException: Array of nodes cannot be empty. Ensure "Rector\DeadCode\Rector\If_\RemoveDeadInstanceOfRector->refactor()" returns non-empty array for Nodes.

A) Return null for no change:

    return null;

B) Remove the Node:

    $this->removeNode($node);
    return $node;
```

Reply:
:+1: